frontends/mit-open/src/pages/FieldPage/FieldPageSkeleton.tsx

```diff
@@ -94,13 +94,18 @@ const FieldSkeletonProps: React.FC<FieldSkeletonProps> = ({
                   display="flex"
                   flexDirection="column"
                   alignItems="start"
-                  sx={{ flexGrow: 24, flexShrink: 0, order: 1, width: "60%" }}
+                  sx={{
+                    flexGrow: 1,
+                    flexShrink: 0,
+                    order: 1,
+                    width: "50%",
+                  }}
                 >
                   <Box
                     display="flex"
                     flexDirection="row"
                     alignItems="center"
-                    sx={{ flexGrow: 1, flexShrink: 0, order: 1, mt: 3 }}
+                    sx={{ flexGrow: 1, flexShrink: 0, order: 1, py: "24px" }}
```

Comment:
For mobile I think this works best as 8px bottom, 0 top (with the 32px bottom padding on the `NavText` noted above.

Reply:
fixed